src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-OverridingInvokersAndRefactoring.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Developer Introduction to the Infusion Framework - Overriding Invokers and Refactoring
+layout: default
+category: Tutorials
+---
+
+With the console and display functionality extracted as a separate components, it's easier to see that large blocks of their configuration are similar:
+
+* they have the same model characteristics
+* they both have a model listener that calls a "say hello" function
+* while their invokers have different names, this isn't necessary now that they've been split out into separate component definitions - we can use invokers to provide each component with an externally identical invoker with a different internal implementation. This is an example of [method overriding](https://en.wikipedia.org/wiki/Method_overriding).
+
+Let's refactor to avoid duplication and create a base "say hello" component that other types of "say hello" component can derive from:
+
+<div class="infusion-docs-note">You can check out the [Live Example](http://codepen.io/waharnum/pen/bgBbOm?editors=1111) of the code below on [CodePen](http://codepen.io/)</div>
+
+```
+fluid.defaults("fluid.helloWorld.sayHello", {
+    gradeNames: ["fluid.modelComponent"],
+    model: {
+        message: "Hello, world!"
+    },
+    modelListeners: {
+        "message": "{that}.sayHello"
```

Comment:
There is a mix of "quoted keys that need to be quoted", "quoted keys that do not need to be quoted", and "unquoted keys" throughout the  examples.

For consistency, I would suggest paring this down to just "keys that need to be quoted" and "unquoted keys".